setup.py

```diff
@@ -35,10 +35,18 @@
         "supervisor.docker",
         "supervisor.addons",
         "supervisor.api",
+        "supervisor.dbus",
```

Comment:
Do we need to list the packages explicitly or can we just find them with `setuptools.find_packages`?